modules/hl7802/cellular_hl7802.c

```diff
@@ -335,6 +335,15 @@ CellularError_t Cellular_ModuleEnableUE( CellularContext_t * pContext )
 {
     CellularError_t cellularStatus = CELLULAR_SUCCESS;
     CellularPktStatus_t pktStatus = CELLULAR_PKT_STATUS_OK;
+    CellularAtReq_t atReqGetWoPrefix =
+    {
+        NULL,
+        CELLULAR_AT_WO_PREFIX,
+        NULL,
+        NULL,
+        NULL,
+        0
+    };
     CellularAtReq_t atReqGetNoResult =
```

Comment:
Do we still need this variable in this function?

Reply:
When modem in echo mode, ATE0 command will be echoed back. This command should be regarded as CELLULAR_AT_WO_PREFIX. Other command will still be regarded as CELLULAR_AT_NO_RESULT command.
```
> ATE0
< ATE0
< OK
```